tests/backup/backup_portworx_test.go

```diff
@@ -562,3 +562,237 @@ var _ = Describe("{RestoreEncryptedAndNonEncryptedBackups}", func() {
+	)
+	labelSelectors := make(map[string]string)
+	cloudCredUIDMap := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	podListBeforeSizeMap := make(map[string]int)
+	podListAfterSizeMap := make(map[string]int)
+
+	var backupLocation string
+	contexts = make([]*scheduler.Context, 0)
+	appNamespaces = make([]string, 0)
+	JustBeforeEach(func() {
+		StartTorpedoTest("ResizeVolumeOnScheduleBackup", "Verify schedule backups are successful while volume resize is in progress", nil, 58050)
+		log.InfoD("Verifying if the pre/post rules for the required apps are present in the list or not")
+		for i := 0; i < len(appList); i++ {
+			if Contains(postRuleApp, appList[i]) {
+				if _, ok := portworx.AppParameters[appList[i]]["post_action_list"]; ok {
+					dash.VerifyFatal(ok, true, "Post Rule details mentioned for the apps")
+				}
+			}
+			if Contains(preRuleApp, appList[i]) {
+				if _, ok := portworx.AppParameters[appList[i]]["pre_action_list"]; ok {
+					dash.VerifyFatal(ok, true, "Pre Rule details mentioned for the apps")
+				}
+			}
+		}
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				appNamespaces = append(appNamespaces, namespace)
+			}
+		}
+	})
+	It("Schedule backup while resizing the volume", func() {
+		providers := getProviders()
+		Step("Validate applications", func() {
+			ValidateApplications(contexts)
+		})
+		Step("Creating rules for backup", func() {
+			log.InfoD("Creating pre and post rule for deployed apps")
+			for i := 0; i < len(appList); i++ {
+				preRuleStatus, ruleName, err := Inst().Backup.CreateRuleForBackup(appList[i], orgID, "pre")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating pre rule for deployed apps for %v with status %v", appList[i], preRuleStatus))
+				dash.VerifyFatal(preRuleStatus, true, "Verifying pre rule for backup")
+				preRuleNameList = append(preRuleNameList, ruleName)
+				postRuleStatus, ruleName, err := Inst().Backup.CreateRuleForBackup(appList[i], orgID, "post")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating post rule for deployed apps for %v with status %v", appList[i], postRuleStatus))
+				dash.VerifyFatal(postRuleStatus, true, "Verifying post rule for backup")
+				postRuleNameList = append(postRuleNameList, ruleName)
+			}
+		})
+		Step("Creating cloud credentials", func() {
+			log.InfoD("Creating cloud credentials")
+			for _, provider := range providers {
+				credName = fmt.Sprintf("%s-%s-%v", "cred", provider, time.Now().Unix())
+				cloudCredUID = uuid.New()
+				cloudCredUIDMap[cloudCredUID] = credName
+				CreateCloudCredential(provider, credName, cloudCredUID, orgID)
```

Comment:
Please refer to PR #1228. CreateCloudCredential now throws an error.

Reply:
updated as per latest code